src/SixLabors.Fonts/GlyphMetrics.cs

```diff
@@ -5,92 +5,105 @@
 using System.Numerics;
 using System.Runtime.CompilerServices;
 using SixLabors.Fonts.Tables.General.Glyphs;
+using SixLabors.Fonts.Unicode;
 
+// TODO: This should support both horizontal and vertical metrics via vmtx table.
 namespace SixLabors.Fonts
 {
     /// <summary>
-    /// A glyph from a particular font face.
+    /// Represents a glyph metric from a particular font face.
     /// </summary>
-    public partial class GlyphInstance
+    public class GlyphMetrics
```

Comment:
I hate to be really picky here but if it does represent a single glyph metric should the class be `GlyphMetric`?

Reply:
Yesn’t…
It’s a single glyph but multiple metric measurements. The generally used term for similar types in this field is the plural.

Reply:
Ah that makes sense thanks